### ISSUE TITLE
aria-hidden="false" is not set to active slides siblings center mode

### ISSUE THREAD

**Issue by asolopovas** [kenwheeler/slick#2812]
Hi something is wrong with the logic when slidesToShow are set to 2,  it does not set area-hidden=true to sibling of two active slides but it only does so when slidesToShow are set to 3 or 4. Basically it keeps 3 elements area-hidden=false rather then two.

http://codepen.io/asolopovas/pen/vxaLVB?editors=0110

**Comment by leggomuhgreggo** [comment]
Adding to the ARIA issues, but it's probably in one of them already. I will make sure when we do a comprehensive a11y fix.

Thanks.

**Comment by jahvi** [comment]
Quick example:
https://codepen.io/javiervd/pen/bWoGOV

Only the middle 2 elements should have a dark grey background but it's applied on 3.

**Comment by kmmathis** [comment]
This seems to be an issue when `slidesToShow` is set to anything greater than 1.

**Comment by kmmathis** [comment]
Sorry, after some more testing it looks like whenever `slidesToShow` is set to an even number, Slick is incorrectly calculating which slides are actually visible.

**Comment by clayton93** [comment]
Is there any update on a fix for incorrect tagging of slide's hidden status for even-numbered `slidesToShow`?

**Comment by maketroli** [comment]
Any updates here? I am having this same issue.

**Comment by laralfield** [comment]
For anyone who needs a fix/hack for this, my solution was to add a very short timeout during `afterChange` to set the `aria-hidden` attribute to `false`. With this bug, Slick Slider takes care of setting each non-active slide back to `true` after each change.

```
$('.slider-name-here').on('afterChange', function(event, slick, currentSlide) {
  setTimeout(function() {
    $('.js-quiz-questions-results .slick-slide.slick-current').attr('aria-hidden', false);
  }, 100);
});
```

**Comment by UmeshSingla** [comment]
From the above suggested fix by @laralfield I modified it to update aria-hidden attribute on load as well. This allows the slide content to be accessible immediately after the load.
`$('.slider__class').on('init afterChange', function(event, slick, currentSlide) {
        setTimeout(function() {
            $('.slick-slide.slick-current .slider__slide').attr('aria-hidden', false);
        }, 100);
    });`

**Comment by ausian** [comment]
// Удаляем атрибут aria-hidden у всех слайдов сразу после инициализации
        $('.cards').find('[aria-hidden]').removeAttr('aria-hidden');

        // Устанавливаем aria-hidden="false" для всех слайдов при смене слайдов
        $('.cards').on('beforeChange', function(event, slick, currentSlide, nextSlide){
            setTimeout(() => {
                slick.$slides.removeAttr('aria-hidden');
            }, 0);
        });
